### ISSUE TITLE
Check if sudo is really needed

### ISSUE THREAD

**Issue by Yenthe666** [Yenthe666/InstallScript#46]
See https://github.com/Yenthe666/InstallScript/blob/9.0/odoo_install.sh#L97
See https://github.com/Yenthe666/InstallScript/pull/24

**Comment by chris001** [comment]
Root (sudo) is only needed for installing the system packages with `apt`, for installing system-wide python packages with `pip`, and for other sys admin actions such as creating the user and adding to group
https://docs.python.org/3.4/installing/index.html
Apart from these, the `odoo-bin` service SHOULD be able to run without issue, under user `odoo`, without the `odoo` user being a member of the `sudoers` group.
During runtime, the `odoo-bin` service should NOT need to perform any privileged actions requiring `sudo` root access.

**Comment by jpaulin** [comment]
By the way, one issue that I'd like to make sure; how about in case where Odoo is set to run in a "low" port, such as 80 (for ordinary web servers)? Still, _installer_ would not need sudo rights?

**Comment by Yenthe666** [comment]
Actually, that should require sudo rights I believe, as the first 1.000+ ports are reserved for root/admin. So that might be a good catch!

**Comment by chris001** [comment]
Yes @jpaulin @Yenthe666 when you run odoo service listening on port <1024 , sudo is required.  This is probably acceptable to develop and test on odoo in a private LAN.
However, in practical usage, Odoo's web server, `werkzeug`, is not a very good idea to expose to the public global internet, so, for best practices, it is smarter to run `nginx` on ports 80 and 443 as reverse `http/https` proxy, and let odoo stay safe and unexposed to the global internet, on port 8069 (or any high port number > 1024).

**Comment by jpaulin** [comment]
@Yenthe666 @chris001 Thanks for clarification, this nailed also the question I'd had for long in my mind: the internal web component of Odoo is thus called "werkzeug".

**Comment by chris001** [comment]
More about werkzeug (German word for "tool"): https://pypi.python.org/pypi/Werkzeug
